src/components/PageElements/Elements/ReviewApplicantResponse.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Icon } from 'semantic-ui-react'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import { ApplicationResponse, ReviewResponse } from '../../../utils/generated/graphql'
+import ApplicantResponseElement from './ApplicantResponseElement'
+import ReviewResponseElement from './ReviewResponseElement'
+import strings from '../../../utils/constants'
+interface ReviewApplicantResponseProps {
+  applicationResponse: ApplicationResponse
+  summaryViewProps: SummaryViewWrapperProps
+  reviewResponse?: ReviewResponse
+  isActiveReviewResponse: boolean
+  isNewApplicationResponse: boolean
+  showModal: () => void
+}
+
+const ReviewApplicantResponse: React.FC<ReviewApplicantResponseProps> = ({
+  applicationResponse,
+  summaryViewProps,
+  reviewResponse,
+  isNewApplicationResponse,
+  isActiveReviewResponse,
+  showModal,
+}) => {
+  const decisionExists = !!reviewResponse?.decision
+  // After review is submitted, reviewResponses are trimmed if they are not changed duplicates
+  // or if they are null, we only want to show reviewResponses that are linked to latestApplicationResponse
+  const isNewReview = applicationResponse.id !== reviewResponse?.applicationResponse?.id
```

Comment:
isActiveReviewResponse ?

Reply:
I have changed in this and `ConsolidationReviewDecision` so the Review element shows if there is some `reviewResponse` received (which is the `thisLatestReviewResponse` only created when the previous check `applicationResponse.id !== reviewResponse?.applicationResponse?.id` was true) and the **Review Button** is the one that display only if the `isActiveReviewResponse` is present, so only the Review button is removed on submitted reviews.